src/Controller/UserController.php

```diff
@@ -57,17 +57,16 @@ public function __construct(LoggerInterface $logger, FormRegistryInterface $form
     public function indexAction(Request $request)
     {
         $table = new EntityTable($this->userService);
-
-        $table->addColumn('user.index.column.id', 'id');
         $table->addColumn('user.index.column.username', 'username');
         $table->addColumn('user.index.column.displayname', 'name');
-        $table->addColumn('user.index.column.email_notification', 'emailNotification', [true => 'fa fa-check-square-o', false => 'fa fa-square-o']);
+        $column = $table->addColumn('user.index.column.email_notification', 'emailNotification', [true => 'fa fa-check-square-o', false => 'fa fa-square-o']);
+        $column->setIconClass('fa fa-bell');
```

Comment:
I see $colum, then $createdColumn, and that one is used multiple times for different colums.

As we are defining stuff here, why not daisy chain the commands? Just as form configuration does?

```suggestion
        $table
            ->addColumn('user.index.column.email_notification', 'emailNotification', [true => 'fa fa-check-square-o', false => 'fa fa-square-o'])
            ->setIconClass('fa fa-bell');
```

Reply:
Agree see my coming PR